tests/utils/testutils_lc.py

```diff
@@ -151,6 +153,15 @@ def _download_azure_functions() -> str:
             with ZipFile(BytesIO(zipresp.read())) as zfile:
                 zfile.extractall(tempfile.gettempdir())
 
+    @staticmethod
+    def _download_extensions() -> str:
+        folder = tempfile.gettempdir()
```

Comment:
if you're using gettempdir() you don't need to assign it to a folder.  you can just  gettempdir() since it will always be the same dir for each run

Reply:
i know its to remove duplicate calls. i can call once and reuse.